source/reference/minio-mc/mc-cat.rst

```diff
@@ -105,22 +105,84 @@ Parameters
       :start-after: start-version-id-desc
       :end-before: end-version-id-desc
 
-.. mc-cmd:: --encrypt-key
+.. mc-cmd:: --enc-kms
+
+   Encrypt or decrypt objects using server-side :ref:`SSE-KMS encryption <minio-sse>` with client-managed keys.
+   
+   The parameter accepts a key-value pair formatted as ``KEY=VALUE``
```

Comment:
note: Assuming @zveinn can identify the root issue w.r.t. the key parser clipping padding bytes.

Also could not figure out how to specify HEX keys, if I can get an example I'll patch those in.